frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -709,6 +710,7 @@ export const experimentLogic = kea<experimentLogicType>([
             const startDate = dayjs()
             actions.updateExperiment({ start_date: startDate.toISOString() })
             values.experiment && eventUsageLogic.actions.reportExperimentLaunched(values.experiment, startDate)
+            activationLogic.findMounted()?.actions.markTaskAsCompleted(ActivationTask.LaunchExperiment)
```

Comment:
Same here, this could be done when experiments are loaded and generally these are loaded after a creation.